join/config.toml

```diff
@@ -396,28 +396,25 @@ namespace = "tendermint"
 [axelar_bridge_btc]
 
 # Address of the bitcoin RPC server
-rpc_addr    = "<your-bitcoin-rpc-and-port>"
+rpc_addr=""
 
 # Timeout period in which the bitcoin node is expected to start (and provide the auth cookie)
 timeout_startup = "60s"
 
 # Timout period after startup in which the bitcoin node is expected to start servicing RPC calls
 timeout_rpc = "100s"
 
-# For testing, the connection to the btc bridge can be ignored.
-# In that case, all functionality requiring access to Bitcoin data is mocked.
-start-with-bridge = true
-
 ##### EVM bridges options #####
 [[axelar_bridge_evm]]
 
 # Chain name
 name = "Ethereum"
-# Address of the ethereum RPC proxy
```

Comment:
We should keep this comment, mirrors bitcoin comment and is consistent

Reply:
done